src/main/java/com/epam/esm/entity/GiftCertificate.java

```diff
@@ -1,33 +1,51 @@
 package com.epam.esm.entity;
 
-import org.springframework.stereotype.Component;
+import com.epam.esm.converter.ZonedDateTimeAttributeConverter;
+import javax.persistence.*;
 import java.math.BigDecimal;
 import java.time.Duration;
 import java.time.ZonedDateTime;
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
 
-@Component
+@Entity(name = "GiftCertificate")
```

Comment:
Why do we need to specify `name = "GiftCertificate"` here?

Reply:
@a1exanddrovich Where is entity name used?

Reply:
made all the names the same in order to avoid confusing